pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -451,6 +451,7 @@ func TestRestartCountByLogDir(t *testing.T) {
 	for _, tc := range []struct {
 		filenames    []string
 		restartCount int
+		isError			 bool
```

Comment:
Would it make sense to have at least one test case with isError = true? Otherwise this field and the assert below look redundant.

Reply:
I don't really want to test the scenario with the wrong folder. I introduced this as I thought we need to test the `strconv` error case, but figured, returning error there doesn't make sense.

Reply:
In this case I'd suggest to remove isError and the code that uses it.